scripts/packages/VSCodeServer/src/trees.jl

```diff
@@ -126,7 +152,8 @@ struct Undef end
 const UNDEF = Undef()
 
 function assign_undefs(xs)
-    xs′ = similar(xs, Any)
+    s = size(xs)
+    xs′ = Array{Any,length(s)}(undef, s...)
```

Comment:
`similar` fails for readonly arrays, so this makes sure we use an  `Array`.

Reply:
Or rather, `similar` of a read-only array produces another read-only array, which then causes a failure on the `setindex!` call below.